curriculum/challenges/english/01-responsive-web-design/registration-form/final.md

```diff
@@ -0,0 +1,190 @@
+---
+id:
+  _bsontype: ObjectID
+  id: "`ð'\t\x9A\x15°\x04\NV=Ó"
```

Comment:
That is quite some `id` @ojeytonwilliams

Reply:
Beautiful.

Reply:
I only just noticed this, sorry.  What on Earth happened here?

Reply:
@ojeytonwilliams I assumed you were to blame... I do not quite understand the significance of a `final.md` file.